src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/scarlet_monastery/boss_mograine_and_whitemane.cpp

```diff
@@ -354,6 +356,16 @@ struct boss_high_inquisitor_whitemaneAI : public CombatAI
         CombatAI::AttackStart(pWho);
     }
 
+    void EnterCombat(Unit* /*enemy*/) override
+    {
+        SetCombatScriptStatus(false);
+        ResetCombatAction(WHITEMANE_ACTION_HEAL, 10000u);
+        ResetCombatAction(WHITEMANE_ACTION_POWERWORD_SHIELD, 15000u);
```

Comment:
You dont need to do these, its automatically called on Evade and on Respawn. Also, this breaks the initial sequence, because you aggro too soon. (she still enters combat, just doesnt react) I.e. the code only must be done on combat after respawn.

Reply:
It plays out properly in game, tho.
But without this, the script would remain disabled after resetting the fight.